database/scripts/lib/report_formatter.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'json'
+
+module ReportFormatter
+  JOB_URL_PATTERN = {
+    /^gz|^sdformat|^ros_gz/ => 'https://build.osrfoundation.org/job/',
+    /^[A-Z]ci/ => 'https://build.ros2.org/job/',
+    /^nightly_/ => 'https://ci.ros2.org/job/',
```

Comment:
What about packaging jobs?

I think this could cause a maintenance burden, can we pull this directly from database/fetcher?

Reply:
The database does not contain any information about the server we're fetching the information.

We can modify the fetcher to add a new `domain` column to `server_status` table. WDYT?

Reply:
That sounds good to me

Reply:
Like we had avoided it, but I think it's worth doing it now that we are favoring pulling the info coming from the dev repos